0_RootFS/gcc_sources.jl

```diff
@@ -236,7 +236,7 @@ function gcc_sources(gcc_version::VersionNumber, compiler_target::Platform; kwar
                 ArchiveSource("https://mirrors.kernel.org/gnu/glibc/glibc-2.19.tar.xz",
                               "2d3997f588401ea095a0b27227b1d50cdfdd416236f6567b564549d3b46ea2a2"),
             ]
-        elseif arch(compiler_target) in ["x86_64", "i686", "powerpc64le"]
+        elseif arch(compiler_target) in ["x86_64", "i686", "powerpc64le", "riscv64"]
```

Comment:
As mentioned in https://github.com/JuliaPackaging/BinaryBuilderBase.jl/pull/375#discussion_r1867458878, this doesn't make sense.  We should use a different glibc version for risc-v, maybe something like 2.36 or so, to target systems we have access to.

Reply:
Thanks. I was trying 2.28; switching to 2.36 now.

Reply:
Hold on a few minutes, I'm trying other RISC-V I have access to, I seem to have different glibc versions there.

Reply:
Ok, some of the boards in https://riscv.epcc.ed.ac.uk/status/ use Ubuntu 22.04 and hence have Glibc 2.35.  Also, according to [Repology](https://repology.org/project/glibc/versions) Red Hat 9 and derivatives (AlmaLinx/Rocky Linux) ship with Glibc 2.34, but my understanding is that Red Hat doesn't currently support RISC-V, so we probably shouldn't care about that at the moment.

To summarise, given the fact we have boards with Glibc 2.35 and Ubuntu 22.04 looks a reasonable baseline for a somewhat recent architecture I think we should target 2.35

Reply:
Using 2.35 now.